src/components/Inputs/Radio/Radio.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { useState } from 'react';
+import { RadioButton } from 'react-native-paper';
+import { QuestionComponentProps } from 'types/types';
+import { TextRegularBold, TextRegular } from 'assets/fonts/Fonts';
+import {
+  ButtonContainer,
+  RadioContainer,
+} from 'components/Inputs/Radio/styles';
+import { TextContainer, TextDescription } from 'components/Inputs/styles';
+
+export default function Radio(props: QuestionComponentProps) {
+  const { question, setAnswer, existingAnswer } = props;
+  const [value, setValue] = useState(existingAnswer);
+
+  const onChange = (val: any): void => {
+    setValue(val);
+    setAnswer(question, val);
+  };
+
+  const getRadioComponent = (option: any): any => {
+    return (
+      <ButtonContainer>
```

Comment:
getting a React unique key error here - this was fixed in earlier PRs, so make sure you're not overriding the change here
<img width="635" alt="Screen Shot 2021-12-05 at 12 23 06 AM" src="https://user-images.githubusercontent.com/63089231/144739207-d3acb841-9894-44a9-8a17-b7e224530259.png">

Reply:
how was it fixed in earlier PRs?

Reply:
https://github.com/calblueprint/siren/blob/125df27f67f0e371a9c183f397ee594e37d0587a/src/components/Radio/radio.tsx#L51